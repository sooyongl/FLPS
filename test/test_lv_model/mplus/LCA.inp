data: file is lca.csv;
variable: names are x1-x10 class;
categorical = x1-x10;
usevariables are x1-x10;
class = c(2);

analysis: type = mixture;

!model:

!%overall%