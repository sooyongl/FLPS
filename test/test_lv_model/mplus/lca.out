Mplus VERSION 8.6
MUTHEN & MUTHEN
11/18/2021   6:35 PM

INPUT INSTRUCTIONS

  data: file is lca.csv;
  variable: names are x1-x10 class;
  categorical = x1-x10;
  usevariables are x1-x10;
  class = c(2);

  analysis: type = mixture;

  !model:

  !%overall%



INPUT READING TERMINATED NORMALLY




SUMMARY OF ANALYSIS

Number of groups                                                 1
Number of observations                                         500

Number of dependent variables                                   10
Number of independent variables                                  0
Number of continuous latent variables                            0
Number of categorical latent variables                           1

Observed dependent variables

  Binary and ordered categorical (ordinal)
   X1          X2          X3          X4          X5          X6
   X7          X8          X9          X10

Categorical latent variables
   C


Estimator                                                      MLR
Information matrix                                        OBSERVED
Optimization Specifications for the Quasi-Newton Algorithm for
Continuous Outcomes
  Maximum number of iterations                                 100
  Convergence criterion                                  0.100D-05
Optimization Specifications for the EM Algorithm
  Maximum number of iterations                                 500
  Convergence criteria
    Loglikelihood change                                 0.100D-06
    Relative loglikelihood change                        0.100D-06
    Derivative                                           0.100D-05
Optimization Specifications for the M step of the EM Algorithm for
Categorical Latent variables
  Number of M step iterations                                    1
  M step convergence criterion                           0.100D-05
  Basis for M step termination                           ITERATION
Optimization Specifications for the M step of the EM Algorithm for
Censored, Binary or Ordered Categorical (Ordinal), Unordered
Categorical (Nominal) and Count Outcomes
  Number of M step iterations                                    1
  M step convergence criterion                           0.100D-05
  Basis for M step termination                           ITERATION
  Maximum value for logit thresholds                            15
  Minimum value for logit thresholds                           -15
  Minimum expected cell size for chi-square              0.100D-01
Optimization algorithm                                         EMA
Random Starts Specifications
  Number of initial stage random starts                         20
  Number of final stage optimizations                            4
  Number of initial stage iterations                            10
  Initial stage convergence criterion                    0.100D+01
  Random starts scale                                    0.500D+01
  Random seed for generating random starts                       0
Link                                                         LOGIT

Input data file(s)
  lca.csv
Input data format  FREE


UNIVARIATE PROPORTIONS AND COUNTS FOR CATEGORICAL VARIABLES

    X1
      Category 1    0.526          263.000
      Category 2    0.474          237.000
    X2
      Category 1    0.526          263.000
      Category 2    0.474          237.000
    X3
      Category 1    0.526          263.000
      Category 2    0.474          237.000
    X4
      Category 1    0.526          263.000
      Category 2    0.474          237.000
    X5
      Category 1    0.526          263.000
      Category 2    0.474          237.000
    X6
      Category 1    0.526          263.000
      Category 2    0.474          237.000
    X7
      Category 1    0.526          263.000
      Category 2    0.474          237.000
    X8
      Category 1    0.526          263.000
      Category 2    0.474          237.000
    X9
      Category 1    0.526          263.000
      Category 2    0.474          237.000
    X10
      Category 1    0.526          263.000
      Category 2    0.474          237.000


RANDOM STARTS RESULTS RANKED FROM THE BEST TO THE WORST LOGLIKELIHOOD VALUES

Final stage loglikelihood values at local maxima, seeds, and initial stage start numbers:

           -1992.698  399671           13
           -1992.698  76974            16
           -1992.698  939021           8
           -1992.698  unperturbed      0



THE BEST LOGLIKELIHOOD VALUE HAS BEEN REPLICATED.  RERUN WITH AT LEAST TWICE THE
RANDOM STARTS TO CHECK THAT THE BEST LOGLIKELIHOOD IS STILL OBTAINED AND REPLICATED.


THE MODEL ESTIMATION TERMINATED NORMALLY



MODEL FIT INFORMATION

Number of Free Parameters                       21

Loglikelihood

          H0 Value                       -1992.698
          H0 Scaling Correction Factor      1.0001
            for MLR

Information Criteria

          Akaike (AIC)                    4027.397
          Bayesian (BIC)                  4115.903
          Sample-Size Adjusted BIC        4049.248
            (n* = (n + 2) / 24)

Chi-Square Test of Model Fit for the Binary and Ordered Categorical
(Ordinal) Outcomes

          Pearson Chi-Square

          Value                            970.763
          Degrees of Freedom                  1002
          P-Value                           0.7550

          Likelihood Ratio Chi-Square

          Value                            305.618
          Degrees of Freedom                  1002
          P-Value                           1.0000



FINAL CLASS COUNTS AND PROPORTIONS FOR THE LATENT CLASSES
BASED ON THE ESTIMATED MODEL

    Latent
   Classes

       1        234.00431          0.46801
       2        265.99569          0.53199


FINAL CLASS COUNTS AND PROPORTIONS FOR THE LATENT CLASSES
BASED ON ESTIMATED POSTERIOR PROBABILITIES

    Latent
   Classes

       1        234.00431          0.46801
       2        265.99569          0.53199


FINAL CLASS COUNTS AND PROPORTIONS FOR THE LATENT CLASSES
BASED ON THEIR MOST LIKELY LATENT CLASS MEMBERSHIP

Class Counts and Proportions

    Latent
   Classes

       1              234          0.46800
       2              266          0.53200


CLASSIFICATION QUALITY

     Entropy                         0.998


Average Latent Class Probabilities for Most Likely Latent Class Membership (Row)
by Latent Class (Column)

           1        2

    1   1.000    0.000
    2   0.000    1.000


Classification Probabilities for the Most Likely Latent Class Membership (Column)
by Latent Class (Row)

           1        2

    1   1.000    0.000
    2   0.000    1.000


Logits for the Classification Probabilities for the Most Likely Latent Class Membership (Column)
by Latent Class (Row)

              1        2

    1      8.241    0.000
    2     -8.441    0.000


MODEL RESULTS

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

Latent Class 1

 Thresholds
    X1$1              -2.169      0.216    -10.056      0.000
    X2$1              -2.167      0.216    -10.050      0.000
    X3$1              -2.169      0.216    -10.054      0.000
    X4$1              -2.170      0.216    -10.058      0.000
    X5$1              -2.167      0.215    -10.058      0.000
    X6$1              -2.169      0.216    -10.053      0.000
    X7$1              -2.169      0.216    -10.056      0.000
    X8$1              -2.167      0.216    -10.052      0.000
    X9$1              -2.168      0.216    -10.050      0.000
    X10$1             -2.169      0.216    -10.056      0.000

Latent Class 2

 Thresholds
    X1$1               2.181      0.203     10.728      0.000
    X2$1               2.179      0.203     10.731      0.000
    X3$1               2.180      0.203     10.731      0.000
    X4$1               2.181      0.203     10.726      0.000
    X5$1               2.179      0.203     10.725      0.000
    X6$1               2.180      0.203     10.728      0.000
    X7$1               2.181      0.203     10.726      0.000
    X8$1               2.179      0.203     10.730      0.000
    X9$1               2.180      0.203     10.733      0.000
    X10$1              2.181      0.203     10.726      0.000

Categorical Latent Variables

 Means
    C#1               -0.128      0.090     -1.429      0.153


QUALITY OF NUMERICAL RESULTS

     Condition Number for the Information Matrix              0.125E+00
       (ratio of smallest to largest eigenvalue)


RESULTS IN PROBABILITY SCALE

                                                    Two-Tailed
                    Estimate       S.E.  Est./S.E.    P-Value

Latent Class 1

 X1
    Category 1         0.103      0.020      5.165      0.000
    Category 2         0.897      0.020     45.203      0.000
 X2
    Category 1         0.103      0.020      5.169      0.000
    Category 2         0.897      0.020     45.132      0.000
 X3
    Category 1         0.103      0.020      5.166      0.000
    Category 2         0.897      0.020     45.185      0.000
 X4
    Category 1         0.103      0.020      5.165      0.000
    Category 2         0.897      0.020     45.227      0.000
 X5
    Category 1         0.103      0.020      5.173      0.000
    Category 2         0.897      0.020     45.166      0.000
 X6
    Category 1         0.103      0.020      5.166      0.000
    Category 2         0.897      0.020     45.181      0.000
 X7
    Category 1         0.103      0.020      5.166      0.000
    Category 2         0.897      0.020     45.205      0.000
 X8
    Category 1         0.103      0.020      5.169      0.000
    Category 2         0.897      0.020     45.153      0.000
 X9
    Category 1         0.103      0.020      5.165      0.000
    Category 2         0.897      0.020     45.155      0.000
 X10
    Category 1         0.103      0.020      5.165      0.000
    Category 2         0.897      0.020     45.204      0.000

Latent Class 2

 X1
    Category 1         0.899      0.019     48.475      0.000
    Category 2         0.101      0.019      5.475      0.000
 X2
    Category 1         0.898      0.019     48.443      0.000
    Category 2         0.102      0.019      5.482      0.000
 X3
    Category 1         0.898      0.019     48.475      0.000
    Category 2         0.102      0.019      5.477      0.000
 X4
    Category 1         0.899      0.019     48.474      0.000
    Category 2         0.101      0.019      5.472      0.000
 X5
    Category 1         0.898      0.019     48.415      0.000
    Category 2         0.102      0.019      5.479      0.000
 X6
    Category 1         0.898      0.019     48.463      0.000
    Category 2         0.102      0.019      5.476      0.000
 X7
    Category 1         0.899      0.019     48.467      0.000
    Category 2         0.101      0.019      5.474      0.000
 X8
    Category 1         0.898      0.019     48.451      0.000
    Category 2         0.102      0.019      5.481      0.000
 X9
    Category 1         0.898      0.019     48.477      0.000
    Category 2         0.102      0.019      5.480      0.000
 X10
    Category 1         0.899      0.019     48.466      0.000
    Category 2         0.101      0.019      5.474      0.000


LATENT CLASS INDICATOR ODDS RATIOS FOR THE LATENT CLASSES

                                                95% C.I.
                    Estimate       S.E.  Lower 2.5% Upper 2.5%

Latent Class 1 Compared to Latent Class 2

 X1
    Category > 1      77.482     22.975     43.330    138.550
 X2
    Category > 1      77.155     22.858     43.170    137.893
 X3
    Category > 1      77.403     22.947     43.292    138.392
 X4
    Category > 1      77.566     22.998     43.380    138.693
 X5
    Category > 1      77.141     22.851     43.166    137.859
 X6
    Category > 1      77.409     22.945     43.298    138.391
 X7
    Category > 1      77.486     22.973     43.337    138.546
 X8
    Category > 1      77.230     22.885     43.207    138.044
 X9
    Category > 1      77.330     22.921     43.255    138.248
 X10
    Category > 1      77.489     22.973     43.339    138.548


DIAGRAM INFORMATION

  Mplus diagrams are currently not available for Mixture analysis.
  No diagram output was produced.


     Beginning Time:  18:35:16
        Ending Time:  18:35:17
       Elapsed Time:  00:00:01



MUTHEN & MUTHEN
3463 Stoner Ave.
Los Angeles, CA  90066

Tel: (310) 391-9971
Fax: (310) 391-8971
Web: www.StatModel.com
Support: Support@StatModel.com

Copyright (c) 1998-2021 Muthen & Muthen
